apps/wpcom-block-editor/src/wpcom/features/tracking/wpcom-block-picker-search-term-handler.js

```diff
@@ -4,14 +4,23 @@
  * External dependencies
  */
 import { debounce } from 'lodash';
+import debug from 'debug';
 
 /**
  * Internal dependencies
  */
 import tracksRecordEvent from './track-record-event';
 
+const tracksDebug = debug( 'wpcom-block-editor:analytics:tracks' );
```

Comment:
Let's replace `tracks` with a context specific to this file, `block-search` or something similar. `tracks` is meant only for the handling of the Tracks integration.

Reply:
let's keep the `track` string there since it belongs to the same context.
It allows seeing all logs about the context, using the wild there.

```js
localStorage.setItem( 'debug', 'wpcom-block-editor:analytics*' )
```

or...

```js
localStorage.setItem( 'debug', 'wpcom-block-editor:analytics:tracks*' )
```

or...

```js
localStorage.setItem( 'debug', 'wpcom-block-editor:analytics:tracks:search' )
```